definitions/npm/xorshift_v1.x.x/flow_v0.32.0-/xorshift_v1.x.x.js

```diff
@@ -0,0 +1,18 @@
+declare module 'xorshift' {
+  declare class $npm$xorshift {
```

Comment:
There is a major refactoring for the official flow doc going on, where I want to add this specific section... for now, you can see most syntax features for module definition in my libdef-talk: https://github.com/ryyppy/flow-libdef-talk/blob/master/dist/flow-libdef-talk.pdf